pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: flutter_nsd
-description: A Flutter plugin that enables Network Service Discovery on various platforms via their native APIs.
-version: 1.3.0
+description: Flutter Network Service Discovery plugin
+version: 1.4.0-alpha01
```

Comment:
Please don't make any changes to this file. It is good as is.

Reply:
reverted in the last commit